### DISCUSSION TITLE
Is there a way to search based on tags?

### DISCUSSION THREAD

**Discussion by MikeTStromberg** [TandoorRecipes/recipes#487] [General]
I can set tags for recipes, which is great, but I don't see how I can search based on those tags, or sort, or do anything with them at all. Not implemented yet, or am I missing something?

**Comment by vabene1111** [comment]
you can open the advanced search (next to the search the three dots), there you can search for tags and ingredients, but to be competently honest the search isn't that great currently except for the name search.

This will definitly be improved in the future

**Reply by tuetenk0pp** [reply]
it would be nice if the search bar would also filter through tags by default

**Reply by vabene1111** [reply]
i am planning on using some kind of input that can search tags, ingredients and free text but i do not yet have the proper technology. But definitely something i want to use at several places all trougout the application
